developer's code of conduct.md

```diff
@@ -16,10 +16,25 @@ To improve review quality, in PR about new feature/feature change/balance change
 5) Be open to criticism. Sometimes the stuff you made and that you are proud of might not be popular in its current form. That's ok and it's a part of the process.
 6) Include a flowchart if it involves adding a new recipe chain. Any new PR without it will be rejected no matter how good it can be. If you don’t know how, try [draw.io](https://draw.io).
 7) Regarding textures within GTNH
-   1) Regarding magic textures: within the vision of the modpack, magic is meant to feel, look, and be different than GregTech, any form of GUI update for the sake of (or that would inadvertently bring) consistency with GT GUIs should be rejected.
+   1) Regarding magic textures: within the vision of the modpack, magic is meant to feel, look, and be different from GregTech, any form of GUI update for the sake of (or that would inadvertently bring) consistency with GT GUIs should be rejected.
    2) Regarding new textures: try to keep textures consistent with the mod/mod group they're being added to.
    3) Regarding tooltips: reduction is fine when needed, but if there is a specific consistency, try to stay within that.
    4) Regarding inventory changing: GUIs that shift the colour of the player's internal inventory are allowed as long as its usage is consistent. Examples being: GT's Steam Machines, Magic Bee's Magic Apiary, Railcraft's Coke Oven, etc.
+8) The design focus of Magic
+   1) As mentioned above, magic within GregTech: New Horizons is meant to have a strong, distinct look, feel, and execution to the typical tech mod in the pack. Thusly, it is of upmost importance that both the artistic and mechanical designs of each mod be treated with the upmost respect and taken into consideration in the creation of new magic content.
+   2) Thaumcraft
+      1) Thaumcraft is a lore mod, while yes, it is a magic mod, above all else it is trying to tell a distinct story, with both limitations and specific paths. As such, when designing and balancing for Thaumcraft, one must take the lore into consideration, one example being: "Essentia types cannot mix otherwise an explosion would occur." Meaning in no circumstance are essentias allowed to touch to maintain canon. (i.e. a single jar can't store multiple essentia but a reservoir can.) However, a designer could take advantage of this and create an essentia explosive, for example writing lore of a mechanism that works similar to a bombardier beetle.
+      2) Thaumcraft lives and dies on the quality of the lore, while there are some allowances for data here and there, hard numbers are meant for the questbook, not the thaumonomicon. 
+      3) Create lore entries with the upmost seriousness, joke entries ***are*** allowed and are encouraged to an extent, however, this is meant for singular isolated items and chains, please do not create an entire research tab for trivial things or jokes.
```

Comment:
maybe delete the part that says "encouraged", if they are sometimes allowed there is no need for them to be encouraged

Reply:
done